packages/renderer/src/lib/ui/KubernetesCheckConnection.svelte

```diff
@@ -11,9 +11,10 @@ import {
 let currentContextName = $derived($kubernetesContexts.find(c => c.currentContext)?.name);
 let error = $state('');
 
-function refresh(): void {
+async function refresh(): Promise<void> {
   error = '';
   if (currentContextName) {
+    await window.telemetryTrack('kubernetes.monitoring.start.current');
```

Comment:
question: wondering here like if we could send if there is a current error on the current state, what was the last time the check occurred, etc

Reply:
@slemeur @Firewall what would be the information of interest, for this telemetry and for the other (when the user asks to start the monitoring of a non-current context)?

Reply:
What @benoitf mentions would definitely be useful. Maybe this is not a property on this event, but knowning the total amount of connections that the user has would also be useful.

Reply:
I'm removing this telemetry, as it is not really related to the PR